include/stl2/detail/range/concepts.hpp

```diff
@@ -159,6 +159,20 @@ STL2_OPEN_NAMESPACE {
 	STL2_CONCEPT ViewableRange =
 		Range<Rng> &&
 		(_RangeImpl<Rng> || View<Rng>);
+
+	namespace ext {
```

Comment:
These were approved by LEWG, right? If so, let's put them directly in `__stl2` and not bother with `ext`.

Reply:
I think everything in P1035 is going back to LEWG for confirmation.

Reply:
Yes, I misremembered the minutes. `ext` it is.